src/commonMain/kotlin/exchange.dydx.abacus/state/manager/Environment.kt

```diff
@@ -505,6 +505,8 @@ class V4Environment(
             )
         }
 
+        private const val DEFAULT_CHAIN_DECIMALS = 18
+        private const val DEFAULT_NON_CHAIN_DECIMALS = 6
```

Comment:
@ruixhuang is there a better name for these constants? Detekt was yelling about this